### PR TITLE
remove lingering bootstrap --branch and -pull

### PR DESCRIPTION
**Pull request by BenTheElder**
fixes https://github.com/kubernetes/test-infra/pull/4569#issuecomment-329566823

### CODE REVIEW

**Comment by krzyzacy** [comment]
bump the gubernator image as well :-)

File: images/pull-test-infra-gubernator/runner [hunk 1]
```diff
@@ -19,8 +19,7 @@ set -o pipefail
 
 git clone https://github.com/kubernetes/test-infra
 ./test-infra/jenkins/bootstrap.py \
-    --repo=k8s.io/${REPO_NAME} \
+    --repo=k8s.io/${REPO_NAME}=${PULL_REFS} \
```
Comment (fejta) [inline images/pull-test-infra-gubernator/runner]: I would suggest removing all these flags from the runner and just relying on $@
all this "help" is a maintenance nightmare.

Comment (BenTheElder) [reply images/pull-test-infra-gubernator/runner]: I agree but I don't actually see where this image is being used yet. @rmmh ?

Comment (BenTheElder) [reply images/pull-test-infra-gubernator/runner]: er nevermind, grepped in the images dir :roll_eyes: I'll do this and bump.

Comment (BenTheElder) [reply images/pull-test-infra-gubernator/runner]: done. bump next

**Comment by BenTheElder** [comment]
noting: `pull-test-infra-gubernator` will keep failing until this goes in.

**Pull request merged**